core/src/main/java/org/eclipse/hono/util/RegistryManagementConstants.java

```diff
@@ -213,6 +213,10 @@
      * all devices to authenticate.
      */
     public static final String FIELD_ADAPTERS_DEVICE_AUTHENTICATION_REQUIRED = "device-authentication-required";
+    /**
+     * The name of the JSON property containing the device identifier.
+     */
+    public static final String FIELD_DEVICE = "device";
```

Comment:
is this field used by the Device Registry Management API types? If not, then I believe it belongs into the MongoDB module as it is implementation specific, isn't it?

Reply:
I have moved this to the utility class inside the MongoDB module.

Reply:
have you?

Reply:
I am sorry. I have moved the fields `updatedOn` and the `version` and somehow missed this. I have changed now and pushed a commit for it.